medicines/api/.env.example

```diff
@@ -1,3 +0,0 @@
-AZURE_SEARCH_INDEX=
-AZURE_SEARCH_KEY=
-AZURE_SEARCH_SERVICE=
```

Comment:
It would still be good to have the `.env.example` file for developers who don't have access to the keyvault we use.

Reply:
Agree.